locales/en.json

```diff
@@ -1,117 +1,135 @@
+  "Your user state has changed.": "Your user state has changed.",
+  "Refresh to load new user state.": "Refresh to load new user state.",
+  "Refresh": "Refresh",
+  "Contacts": "Contacts",
+  "Report an issue": "Report an issue",
+  "Meet us on %s": "Meet us on %s",
+  "Send us email": "Send us email",
+  "Documents": "Documents",
+  "Features": "Features",
+  "YAML Metadata": "YAML Metadata",
+  "Slide Example": "Slide Example",
+  "Cheatsheet": "Cheatsheet",
+  "Example": "Example",
+  "Syntax": "Syntax",
+  "Header": "Header",
+  "Unordered List": "Unordered List",
+  "Ordered List": "Ordered List",
+  "Todo List": "Checklist",
+  "Blockquote": "Blockquote",
+  "Bold font": "Bold",
+  "Italics font": "Italicize",
+  "Strikethrough": "Strikethrough",
+  "Inserted text": "Underlined text",
+  "Marked text": "Highlighted text",
+  "Link": "Link",
+  "Image": "Image",
+  "Code": "Code",
+  "Externals": "Externals",
+  "This is a alert area.": "This is an alert area.",
+  "Revert": "Revert",
+  "Import from clipboard": "Import from clipboard",
+  "Paste your markdown or webpage here...": "Paste your markdown or webpage here…",
+  "Clear": "Clear",
+  "This note is locked": "This note is locked",
+  "Sorry, only owner can edit this note.": "Sorry, only the owner can edit this note.",
+  "OK": "OK",
+  "Reach the limit": "Reach the limit",
+  "Sorry, you've reached the max length this note can be.": "Sorry, you've reached the maximum length this note can be.",
+  "Please reduce the content or divide it to more notes, thank you!": "Please shorten the note.",
+  "Import from Gist": "Import from Gist",
+  "Paste your gist url here...": "Paste your gist url here…",
+  "Import from Snippet": "Import from Snippet",
+  "Select From Available Projects": "Select From Available Projects",
+  "Select From Available Snippets": "Select From Available Snippets",
+  "OR": "OR",
+  "Export to Snippet": "Export to Snippet",
+  "Select Visibility Level": "Select Visibility Level",
+  "Night Theme": "Night Theme",
+  "Follow us on %s and %s.": "Follow us on %s, and %s.",
+  "Privacy": "Privacy",
+  "Terms of Use": "Terms of Use",
+  "Do you really want to delete your user account?": "Do you really want to delete your user account?",
+  "This will delete your account, all notes that are owned by you and remove all references to your account from other notes.": "This will delete your account, all notes that are owned by you and remove all references to your account from other notes.",
+  "Delete user": "Delete user",
+  "Export user data": "Export user data",
+  "Help us translating on %s": "Help us translating on %s",
+  "Account Settings for %s": "Account Settings for %s",
+  "Account Settings": "Account Settings",
+  "About You": "About You",
+  "Visit public profile": "Visit public profile",
+  "User could not be saved.": "User could not be saved",
+  "name/Validation is failed": "Name contains invalid characters",
```

Comment:
In two places I did not conform to the normal "human readable" pattern of this file. This is the first: the validation error messages from sequelize are just bad. To know which field is affected, I concatenate them with a slash.